source/braille.py

```diff
@@ -27,73 +27,181 @@
+	# check box.
+	controlTypes.ROLE_CHECKBOX: _("chk"),
+	# Translators: Displayed in braille for an object which is a
+	# radio button.
+	controlTypes.ROLE_RADIOBUTTON: _("rbtn"),
 	# Translators: Displayed in braille for an object which is an
 	# editable text field.
 	controlTypes.ROLE_EDITABLETEXT: _("edt"),
 	# Translators: Displayed in braille for an object which is a
-	# list.
-	controlTypes.ROLE_LIST: _("lst"),
+	# button.
+	controlTypes.ROLE_BUTTON: _("btn"),
 	# Translators: Displayed in braille for an object which is a
 	# menu bar.
 	controlTypes.ROLE_MENUBAR: _("mnubar"),
 	# Translators: Displayed in braille for an object which is a
+	# menu item.
+	controlTypes.ROLE_MENUITEM: _("mnuitem"),
+	# Translators: Displayed in braille for an object which is a
 	# menu.
 	controlTypes.ROLE_POPUPMENU: _("mnu"),
 	# Translators: Displayed in braille for an object which is a
-	# button.
-	controlTypes.ROLE_BUTTON: _("btn"),
+	# combo box.
+	controlTypes.ROLE_COMBOBOX: _("cbo"),
 	# Translators: Displayed in braille for an object which is a
-	# check box.
-	controlTypes.ROLE_CHECKBOX: _("chk"),
+	# list.
+	controlTypes.ROLE_LIST: _("lst"),
 	# Translators: Displayed in braille for an object which is a
-	# radio button.
-	controlTypes.ROLE_RADIOBUTTON: _("rbtn"),
+	# graphic.
+	controlTypes.ROLE_GRAPHIC: _("gra"),
 	# Translators: Displayed in braille for an object which is a
-	# combo box.
-	controlTypes.ROLE_COMBOBOX: _("cbo"),
+	# help balloon.
+	controlTypes.ROLE_HELPBALLOON: _("hlp"),
+	# Translators: Displayed in braille for an object which is a
+	# tool tip.
+	controlTypes.ROLE_TOOLTIP: _("tltip"),
 	# Translators: Displayed in braille for an object which is a
 	# link.
 	controlTypes.ROLE_LINK: _("lnk"),
 	# Translators: Displayed in braille for an object which is a
-	# dialog.
-	controlTypes.ROLE_DIALOG: _("dlg"),
-	# Translators: Displayed in braille for an object which is a
 	# tree view.
 	controlTypes.ROLE_TREEVIEW: _("tv"),
 	# Translators: Displayed in braille for an object which is a
+	# tree view item.
+	controlTypes.ROLE_TREEVIEWITEM: _("tvitem"),
+	# Translators: Displayed in braille for an object which is a
+	# tab control.
+	controlTypes.ROLE_TABCONTROL: _("tabctl"),
+	# Translators: Displayed in braille for an object which is a
+	# progress bar.
+	controlTypes.ROLE_PROGRESSBAR: _("prgbar"),
+	# Translators: Displayed in braille for an object which is a
+	# scroll bar.
+	controlTypes.ROLE_SCROLLBAR: _("scrlbar"),
+	# Translators: Displayed in braille for an object which is a
+	# status bar.
+	controlTypes.ROLE_STATUSBAR: _("stbar"),
+	# Translators: Displayed in braille for an object which is a
 	# table.
-	controlTypes.ROLE_TABLE: _("tb"),
+	controlTypes.ROLE_TABLE: _("tbl"),
+	# Translators: Displayed in braille for an object which is a
+	# tool bar.
+	controlTypes.ROLE_TOOLBAR: _("tlbar"),
 	# Translators: Displayed in braille for an object which is a
+	# drop down button.
+	controlTypes.ROLE_DROPDOWNBUTTON: _("ddbtn"),

```

Comment:
I wonder if drbtn might be a bit more intuitive? I know dd stands for dropdown, but when all else fails, I try to sound it out. Drbtn seems to click more quickly in my head because of the "dr" sound, but I get that this is subjective.